test/integration/package/controls/config.rb

```diff
@@ -11,4 +11,11 @@
     its('content') { should include 'DOCKER_OPTS="-s btrfs --dns 8.8.8.8"' }
     its('content') { should include 'export http_proxy="http://172.17.42.1:3128"' }
   end
+  describe file('/etc/docker/daemon.json') do
+    it { should be_file }
+    its('owner') { should eq 'root' }
+    its('group') { should eq 'root' }
+    its('mode')  { should cmp '0640' }
+    its('content') { should include '"insecure-registries": []'}
```

Comment:
We've got a [`rubocop` violation](https://gitlab.com/saltstack-formulas/docker-formula/-/jobs/1205924726#L117) that needs to be fixed here:

```suggestion
    its('content') { should include '"insecure-registries": []' }
```

Reply:
Thanks. Rebase is done and applied this at the same time.